sumologic/resource_sumologic_generic_polling_source.go

```diff
@@ -130,6 +130,23 @@ func resourceSumologicGenericPollingSource() *schema.Resource {
 						},
 					},
 				},
+				"sns_topic_arn": {
```

Comment:
suggest changing to `sns_topic_or_subscription_arn` to stay consistent with API because it could be either one
```
// NOTE(srishti, 2018-06-06): If successful, the subscription ARN is stored, topic ARN otherwise
case class SnsTopicOrSubscriptionArn(isSuccess: Boolean, arn: String)
```